src/tutorials/media_library.md

```diff
@@ -0,0 +1,221 @@
+
+* **Completion Time:** ~20 minutes
+* **Knowledge Level:** No previous knowledge.
+* **Skills you will learn:**
+    * Where to find the Media Library.
+    * How to upload images to the Media Library.
+    * Creating image brushes from content in your Media Library.
+    * Using image brushes in the UI to create a shop.
+
+## Download Image Pack
+
+For this tutorial, you can use your images if you prefer, or if you want to follow along to recreate the shop UI, then you will need to download an image pack such as the one linked below. The pack below will require you to cut or slice the parts you want to use.
+
+[Mobile Game GUI](https://graphicburger.com/mobile-game-gui/)
+
+Always check the license that comes with any images you download that you will be using in your project. For the above pack, the license is included with the download.
+
+## Uploading Images to the Media Library
+
+To use images in any of your projects, the images need to be uploaded to your Media Library account first. Once the images have been uploaded, you can then create brushes that can be used in the UI.
+
+### Open Media Library
+
+To open the Media Library, go to the **Window** menu at the top of your Core Editor and select **Media Library** from the list.
+
+![!Window Menu](../img/MediaLibraryTutorial/window_menu.png){: .center loading="lazy" }
+
+### Uploading Images
+
+From the Media Library window, select the **Upload Image** button and upload all the images you want to use for your UI.
+
+In the image below, all the images that will be used in this tutorial have been uploaded.
+
+![!Media Library](../img/MediaLibraryTutorial/media_library.png){: .center loading="lazy" }
+
+## Creating Brushes
+
+Before you can use an image in your project, a custom brush needs to be created. A brush is a usable asset that references the image on your account.
+
+For each image in the Media Library you want to use in your UI, do the following:
+
+1. Click on an image in the Media Library.
+2. Click on the **Create Image Asset** button.
+
+When clicking the **Create Image Asset** button, a new brush will be created in the folder **My Brushes** in the **Project Content** window.
+
+![!Create Brushes](../img/MediaLibraryTutorial/my_brushes.png){: .center loading="lazy" }
+
+## Creating the UI
+
+Now that you have the brushes created, you can start putting together the UI by dragging the brushes into the **Hierarchy**. If you are using your own images, then the placement and scale of your images will be different.
+
+### Understanding Aspect Ratio
+
+When using custom images, they will look better when you match their aspect ratio. Each UI Image in the Hierarchy has an option in the Properties window to retain the aspect ratio. Aspect ratio is a proportional relationship between the width and height of an image.
+
```

Comment:
Can you add an image for this? Before and after and a picture of the property in the Properties window?